graylog2-server/src/test/java/org/graylog/scheduler/DBJobTriggerServiceTest.java

```diff
@@ -319,10 +319,7 @@ public void updateTrigger() {
         final JobTriggerDto originalTrigger = dbJobTriggerService.create(JobTriggerDto.Builder.create(clock)
                 .jobDefinitionId("abc-123")
                 .jobDefinitionType("event-processor-execution-v1")
-                .schedule(IntervalJobSchedule.builder()
-                        .interval(15)
-                        .unit(TimeUnit.SECONDS)
-                        .build())
+                .schedule(OnceJobSchedule.create())
                 .build());
```

Comment:
Please let me know why you swapped the schedule type with the updated trigger below.

Reply:
See description for https://github.com/Graylog2/graylog2-server/pull/18946/commits/a951dd6f52a63e3559c094dae514d080b23b56ce

Reply:
To elaborate further, the `OnceJobSchedule` doesn't have any fields besides the type name, which all other schedule types also have. When serializing the update operation, the schedule would lose the type-specific fields. 

The test wasn't covering that because we would update an `IntervalJobSchedule` with a `OnceJobSchedule` that only has the type field anyway. Serializing that would work just fine. However, if we use an `IntervalJobSchedule` in the update operation, we should create an object that couldn't be deserialized properly in the old version without the now-removed workaround.